codemap/api-hot.md

```
@@ -325,22 +325,22 @@ when WRITING TESTS; they are not feature-code frequency signal.
 - `encodeText :: ToJSON value => value -> Text`  <!-- 117 test call sites -->
 - `object :: [(Text, Value)] -> Value`  <!-- 105 test call sites -->
 - `null :: Value`  <!-- 54 test call sites -->
-- `encode :: ToJSON value => value -> Value`  <!-- 44 test call sites -->
-- `decode :: FromJSON value => Value -> Result Text value`  <!-- 24 test call sites -->
+- `encode :: ToJSON value => value -> Value`  <!-- 45 test call sites -->
+- `decode :: FromJSON value => Value -> Result Text value`  <!-- 25 test call sites -->
 - `withObject :: Text -> (Object -> Parser value) -> Value -> Parser value`  <!-- 3 test call sites -->
 
 ## ConcurrentVar
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)